WordPressLint/src/test/java/org/wordpress/android/lint/MissingNullAnnotationDetectorTest.kt

```diff
@@ -267,7 +267,7 @@ class MissingNullAnnotationDetectorTest {
               @Inject ExampleClass(String name) {}
             }
         """).indented())
-                .issues(MissingNullAnnotationDetector.MISSING_METHOD_PARAMETER_ANNOTATION)
+                .issues(MissingNullAnnotationDetector.MISSING_CONSTRUCTOR_PARAMETER_ANNOTATION)
```

Comment:
Praise (❤️): How did I miss this in my previous review, my apologies and kudos for fixing it! 😅 🤷 😅

Reply:
Probably the same way that I missed it: most of that line is _almost_ the same 😅 . The good thing is, you discovered it via the tool usage, and reported it. 🥇